### ISSUE TITLE
Do not support Wiki tags on Textbox

### ISSUE THREAD

**Issue by eduardomozart** [splitbrain/dokuwiki-plugin-bureaucracy#335]
When creating the following form:

```
action template :sw:mellanox:mlag:template :sw:mellanox:mlag :

fieldset "Switch 01 (Master)"
textbox "Hostname do Switch 01" "=sx01" /^(?![0-9]+$)(?!.*-$)(?!-)[a-zA-Z0-9-]{1,63}$/ "**Digite um hostname válido para o Switch 01 (Master). Para estar em conformidade com a RFC [[rfc>1123]], o hostname deve conter apenas letras, números e hífens e não deve começar ou terminar com um hífen."

fieldset
submit "Criar Página"
```

Clicking on submit, the ``[[rfc>1123]]`` tag on TextBox error text message when RegEx doesn't match is shown as is instead of rendering a link to RFC 1123.

**Comment by Michaelsy** [comment]
What makes you think you can use wiki syntax in the error message? I don't think that's the case.

\- Michael Sy.

**Comment by eduardomozart** [comment]
I believe it should be. Any Wiki syntax should be rendered as such, it should be rendered on server side first and then loaded to the error message. There's no statement on documentation that errors can't use Wiki syntax.

**Comment by tomio13** [comment]
The developer guide of dokuwiki states it clearly that calling the renderer from within a plugin is a computationally expensive operation.

**Comment by Michaelsy** [comment]
> There's no statement on documentation that errors can't use Wiki syntax.

You don't really want to have an endless list of all the functions in the documentation that are not available - do you?

Basically, I like the idea of being able to use links in error messages, as this allows the user to be quickly referred to further information. You don't have to support the entire wiki syntax for this. 

I'm just afraid that such a feature request is too far down the list of priorities to be realised in the foreseeable future.

\- Michael Sy.

**Comment by eduardomozart** [comment]
No, but the most common ones should be there. We can use Wiki text format on forms description, so alerts could work, too. Maybe using a custom jQuery alert would allow such syntax be used as those links could be useful to the end user. I'm not sure, but maybe there's some simple rendering available on JS side, like replacing links on it's markdown syntax to the expected ones, or maybe even using a custom jQuery alert would allow HTML syntax ones. But anyway, it isn't a priority, just a feature suggestion.

**Comment by Michaelsy** [comment]
>  calling the renderer from within a plugin is a computationally expensive operation.

Yes, that is correct. But what does "expensive" actually mean in this case? The only time I had problems with this was when I put the Wst plugin on a wiki page about 100 times (the Wst plugin interprets wiki syntax). This gave me a response time of about 15s. In this case, "expensive" means a computing effort of 150 ms. (DokuWiki on a Stick, Greebo, PHP 5.6, old Win computer) This is bearable if it only occurs a few times on a page.

Incidentally, in the Bureaucracy plugin there is the wiki field in which wiki syntax is also interpreted and is therefore subject to the same problem. But if used sparingly, this is not an issue.

\- Michael Sy.
